.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -41,6 +41,10 @@ or from https://gitmoji.dev/
 
 <!-- Give REVIEWERS some hits or code snippets on how could this be tested -->
 
+## Dev Checklist
+
+- [ ] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))
```

Comment:
Why not just putting it in the existing devops checklist?

Reply:
lte's try it this way and then see